.github/workflows/wheels.yaml

```
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu-20.04", "macos-latest" ]
-        python: [ "3.8" , "3.9", "3.10" ]
+        python: [ "3.8" , "3.9", "3.10", "3.11" ]
```

setup-python needs the `-dev` suffix for pre-release versions of Python (there is a discussion about this going on now).
```suggestion
        python: [ "3.8" , "3.9", "3.10", "3.11-dev" ]
```

---

This isn't needed for our macos or linux wheel builds because the Python version is managed by multibuild, not a GitHub action.

@EwoutH this is a project with a lot of legacy and not a lot of documentation, I would be surprised if you understood it completely :smile:

---

A that explains it, thanks. I saw the failure on Windows and assumed this would be also the case here